contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,78 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## TL;DR
+
+The patch release czar will summarize all issues discovered since the last patch release weekly
+at the SIG release meeting and publish the summary to kubernetes-sig-release@googlegroups.com.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating critical fixes to the community
+
+In order to provide better transparency with the community on the patch release status and process,
+the patch release czar provides a weekly summary of all issues that have been discovered since the
+*last patch release* (for the most recent minor release only).
+
+The summary will be presented at the weekly sig-release meeting as the first agenda item (recorded
```

Comment:
Agreed it is too much to have the patch manager attend both meetings and give summaries in both.  Is the community meeting still the correct place to give a release summary?  Should we just move it to the sig-release meeting and send out an update to kubernetes-dev?